sdk/python/feast/registry.py

```diff
@@ -65,6 +66,16 @@
     "": "LocalRegistryStore",
 }
 
+REGISTRY_OBJECT_TYPE_TO_STR = {
```

Comment:
we want to convert plural -> single, so removing underscores doesn't quite do it

Reply:
I see.. this is because we use them directly in feast plan output to the console.

Mind making a comment explaining why this mapping exists? Also, it may make more sense to store this map closer to where it's used (i.e. near some feast plan file or in what today is `repo_operations`)